packages/components/src/components/atoms/BaseCard/BaseCard.tsx

```diff
@@ -1,19 +1,37 @@
 /* eslint-disable max-lines */
-import { CardHandle } from '@models/CardHandle';
-import { CardProps } from '@models/CardProps';
-import { Position } from '@models/Position';
-import classnames from 'classnames';
-import React, { forwardRef, KeyboardEvent, useImperativeHandle, useState, useEffect, useCallback } from 'react';
+import React, { FunctionComponent } from 'react';
+import { Draggable } from '@templates/Draggable';
+import classNames from 'classnames';
 import { createUseStyles } from 'react-jss';
+import { Dimensions } from '@models/Dimensions';
+import { Position } from '@models/Position';
 
 /**
  * Properties for BaseCard component
  */
-export interface BaseCardProps extends CardProps {
+export interface BaseCardProps {
```

Comment:
Any reason why it does not extends anymore `CardProps`?

Reply:
Because I removed `CardProps`, we don't use it anymore since we don't have `PlayingCard` component anymore 🙃

Reply:
Yes indeed 😓 I removed them